### DISCUSSION TITLE
First Time using Gantt

### DISCUSSION THREAD

**Discussion by najibnasich** [dotnetcore/BootstrapBlazor#3917] [Q&A]
Hi Argo and other developer,
thank you for creating this awesome framework. I am currently using this template and trying to make gantt chart. I am a beginner of dotner programmer. Please advise me about gantt chart, because I try to make it by your code reference but found this:

error CS0246: The type or namespace name 'GanttItem' cou
ld not be found

**Comment by Vision-Zhang** [comment]
Have you installed the Gantt extension package？

**Reply by najibnasich** [reply]
No, I dont install it yet. Which package that I have to install? I dont know the name of the package
